apps/website/src/content/docs/skiptocontentlink.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Skip to content link
+description: A skip-to-content link can be used to navigate directly to certain desired contents.
+thumbnail: #TODO
+---
+
+<p>{frontmatter.description}</p>
+
+<Placeholder componentPageName='table--basic' />
+
+## Usage
+
+`SkipToContentLink` is designed for screen reader and keyboard users and will only become visible when focused. It provides a shortcut to the main content of the page, bypassing the header and other navigation elements. The `href` prop should be set to the `id` of the main content section, and the value must include the '#' prefix.
```

Comment:
The first two sentences here do not belong under "Usage", they should be placed in the intro section.

Reply:
I have moved these in the intro section.